backend/routes/listing.js

```diff
@@ -149,23 +123,21 @@ async function getListings(
   let lat = 0;
   let lng = 0;
 
-  console.log(latlng);
-  console.log(sanitize(latlng));
-  console.log(sanitize(latlng).split(","));
   try {
-    lat = sanitize(latlng).split(",")[0];
-    lng = sanitize(latlng).split(",")[1];
+    lat = parseFloat(sanitize(latlng).split(",")[0]);
+    lng = parseFloat(sanitize(latlng).split(",")[1]);
   } catch {}
 
-  console.log(lat);
-  console.log(lng);
-
   /* These are all query filters */
   if (title && title !== "") {
     match.push({ title: new RegExp(sanitize(title), "i") });
   }
   if (claimed) {
-    match.push({ claimed: claimed });
+    if (claimed === "true") {
+      match.push({ claimed: true });
+    } else {
+      match.push({ claimed: false });
+    }
   } else {
     match.push({ claimed: false });
   }
```

Comment:
Shouldn't we just not push a claimed filter if it is not set to true?
So
`
  if (claimed) {
    match.push({ claimed: true });
  }
`

Reply:
Good idea!

Reply:
if, else isn't necessary here. `match.push({ claimed: claimed })` does the same thing